JavaScript/common.js

```diff
@@ -1,6 +1,32 @@
+// Drop down menu functionality
 const nav_button = document.getElementById("nav_button");
 const nav_bar = document.getElementById("nav_bar");
 
 nav_button.addEventListener('click', () => {
-    nav_bar.classList.toggle('hidden');
-});
\ No newline at end of file
+  nav_bar.classList.toggle('hidden');
+});
+
+// Geolocation Api
+const findState = () => {

```

Comment:
Could use a better name. State can refer to a state in the US or an object state or something similar